TWLight/applications/views.py

```diff
@@ -982,11 +981,10 @@ def post(self, request, *args, **kwargs):
         # on their valid authorizations to ensure we have enough accounts available and set the
         # boolean value for the corresponding partner_pk in the partners_distribution_flag dictionary.
         for partner_pk, app_count in applications_per_partner.items():
-            total_accounts_available = Partner.objects.filter(pk=partner_pk).values(
-                "accounts_available"
-            )[0]["accounts_available"]
```

Comment:
I may have been missing something here but this seemed to be an extremely roundabout way of retrieving one value from one object.